spisea/atmospheres.py

```diff
@@ -318,9 +318,9 @@ def get_cmfgenRot_atmosphere_closest(metallicity=0, temperature=24000, gravity=4
     """
     # Set up the proper root directory
     if rebin == True:
-        root_dir = '/g/lu/models/cdbs/grid/cmfgen_rot_rebin'
+        root_dir = '/g/lu/models/cdbs_20/grid/cmfgen_rot_rebin'
```

Comment:
If we want to keep this function, we should remove these hard-coded paths and change them to call the environment variable PYSYN_CDBS. The PYSYN_CDBS value should replace "/g/lu/models/cdbs".